src/main/java/io/supertokens/storage/mongodb/Start.java

```diff
@@ -380,4 +388,41 @@ public boolean setJWTSigningKeyInfoIfNoKeyForAlgorithmExists_Transaction(JWTSign
             throw new StorageQueryException(e);
         }
     }
+
+    @Override
+    public String getUserPoolId() {
+        // we do not allow multiple in memory dbs as that is not really useful in any way..
+        return "same-user-pool";
+    }
+
+    @Override
+    public String getConnectionPoolId() {
+        // we do not allow multiple in memory dbs as that is not really useful in any way..
```

Comment:
wrong comment - please search for the word memory

Reply:
Done